openedx/core/djangoapps/content/learning_sequences/views.py

```diff
@@ -108,14 +112,20 @@ def _sequence_repr(self, sequence, sequence_schedule, accessible_sequences):
                     'due': sequence_schedule.due,
                 }
 
-            return {
+            sequence_representation = {
                 "id": str(sequence.usage_key),
                 "title": sequence.title,
                 "accessible": sequence.usage_key in accessible_sequences,
                 "inaccessible_after_due": sequence.inaccessible_after_due,
                 **schedule_item_dict,
             }
 
+            # Only include this data if special exams are on
+            if settings.FEATURES.get('ENABLE_SPECIAL_EXAMS', False):
+                sequence_representation["exam_information"] = exam_information
```

Comment:
The fields in the `sequence_representation` above have the same names as those in `CourseLearningSequenceData`. Can we also call this `exam` for consistency ("information" is kind of redundant)?